src/components/common/ColonyHome/ColonyMembersWidget/MembersSubsection.tsx

```diff
@@ -129,10 +123,10 @@ const MembersSubsection = ({
         )}
       </div>
     ),
-    [isContributorsSubsection, membersPageRoute, colonyWatchers, name],
+    [isContributorsSubsection, membersPageRoute, members, name],
   );
 
-  if (!colonyWatchers.length) {
+  if (!members.length) {
```

Comment:
This will be true if the members array is empty, which is expected to happen and is not necessarily an error. We shouldn't be showing the error message here. 

Can I suggest that you pass the `members` prop with its original type (including null and undefined) and if it's null or undefined, then show the error here. If it's just empty, then we shouldn't show anything

Reply:
Thanks @jakubcolony for all your comments and suggestions. They are all resolved now 😉.